### PR TITLE
LZ Guide Improvements

### PR DESCRIPTION
**Pull request by rhoboat**
Friend of #441, #442.

Include a link to service catalog.
Add more stuff to common.hcl.

### CODE REVIEW

File: _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc [hunk 1]
```diff
@@ -1027,6 +1028,11 @@ locals {
   # TODO: Enter the default AWS region, the same as where the terraform state S3 bucket is currently provisioned.
   default_region = ""
 
+  # TODO: Fill these in after applying the account-baseline-root to the root account.
+  config_s3_bucket_name = ""
+  cloudtrail_s3_bucket_name = ""
+  cloudtrail_kms_key_arn = ""
```
Comment (brikis98) [inline _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc:1034]: Couldn't you fill in the bucket names _now_?

Also, NIT: `fmt` the code :)

Comment (rhoboat) [reply _posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc:1034]: Is this the recommended approach? AFAIK these are optional inputs to the account-baseline-root, which we don't guide them to set in this guide.

**Review by ina-stoyanova — APPROVED** [review]
Looks good to me! 

Just to double-check: Is this ready to be merged? I don't believe there are any outstanding tasks based on previous comments.

**Comment by rhoboat** [comment]
I am not sure if the bucket names should be filled in now, but based on my judgment I think we're good to merge this.

**Comment by rhoboat** [comment]
Thanks for the review!

**Pull request merged**